single_kernel_mongo/managers/mongos_operator.py

```diff
@@ -326,17 +333,23 @@ def proxy_information_to_client_and_handle_connectivity(self, relation: Relation
 
     # BEGIN: Helpers
     def update_external_services(self):
```

Comment:
sorry nit here, but can we rename to `update_k8s_external_services`, I say this because the mongos vm service can be made external, but this function is purely for k8s